php-nginx/build-scripts/install_php56.sh

```diff
@@ -40,6 +40,8 @@ apt-get install -y \
         --no-install-recommends
 
 # Enable some extensions for backward compatibility
+ln -s ${PHP56_DIR}/bin/php56-enmod ${PHP56_DIR}/bin/php-enmod
```

Comment:
No action required: If we don't use these command any more, maybe we can eventually stop creating these links, also remove them from the debian packages.